src/main/java/com/clussmanproductions/trafficcontrol/blocks/BlockTrafficLightControlBox.java

```diff
@@ -11,18 +12,30 @@
 import net.minecraft.block.properties.PropertyDirection;
 import net.minecraft.block.state.BlockStateContainer;
 import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.Minecraft;
```

Comment:
I see we're adding a client-only class here.  This might not be a problem as I'm pretty sure Forge lazy-loads the classes, but make sure this works on a dedicated server as well.

Reply:
Server-side stability not impacted.